src/locales/en.json

```diff
@@ -116,7 +116,7 @@
         "warning": "No backup codes at this time, please create some."
       },
       "dndCard": {
-        "title": "Do not disclose",
+        "title": "Pretend I don't exist",
```

Comment:
Another wording that could be a good option is "Hide my account", but if "Pretend I don't exist" has already been settled on, that's fine too.